cmd/earthly/org_cmds.go

```diff
@@ -281,23 +278,27 @@ func (app *earthlyApp) actionOrgInviteEmail(cliCtx *cli.Context) error {
 		OrgName: orgName,
 	}
 
-	permission := cliCtx.String("permission")
+	permission := app.invitePermission
 	if permission == "" {
-		permission, err = promptInput(cliCtx.Context, "New user's permission (read, write, admin): ")
+		permission, err = promptInput(cliCtx.Context, "New user's permission [read/write/admin] (default=read): ")
 		if err != nil {
 			return errors.Wrap(err, "failed to read permission")
 		}
 	}
-	invite.Permission = permission
-
-	message := cliCtx.String("message")
-	if message == "" {
-		message, err = promptInput(cliCtx.Context, "Message to user: ")
-		if err != nil {
-			return errors.Wrap(err, "failed to read message")
-		}
+	permission = strings.ToLower(permission)
+	switch permission {
+	case "":
+		permission = "read"
+	case "r":
```

Comment:
Are we expecting the first char or the full word? Based on the prompt text I'd assume the latter.

Reply:
Perhaps it should take both?

Reply:
This simply allows it to take both

Reply:
I suppose with the `default` it'll pass through the full word. Might be nice to limit/validate, but this is fine.

Reply:
:+1: Added validation